README.md

```diff
@@ -150,7 +150,7 @@ You can also set up the application using `virtualenv` by doing the following:
 To run the test suite, use the following commands:
 
 ```sh
-coverage manage.py test -v 3
+coverage run manage.py test -v 3
```

Comment:
what was the reason for adding `run`?

Reply:
I don't think it works if you don't have that. This is the command we are using in `start.sh`.
https://github.com/tl-its-umich-edu/placement-exams/blob/920c962d83b9473195b9b24d4c85f11a19c708c8/start.sh#L19-L22

Reply:
OK, it's just a fix the doc with correct info then

Reply:
Yup, I think I'll add `coverage report`, too.